packages/react-reconciler/src/ReactFiberUnwindWork.new.js

```diff
@@ -108,20 +104,18 @@ function unwindWork(
     }
     case SuspenseComponent: {
       popSuspenseContext(workInProgress);
-      if (enableSuspenseServerRenderer) {
```

Comment:
I suppose some of these blocks wrapped by a `supportsHydration` flag instead to reduce to bundle size for non-DOM renderers